cypress_test/integration_tests/common/helper.js

```diff
@@ -375,11 +375,20 @@ function documentChecks(skipInitializedCheck = false) {
 						});
 				});
 		});
+
+		// In Writer wait for styles to appear in notebookbar
+		doIfOnDesktop(() => {
+			doIfInWriter(() => {
+				cy.cGet('#stylesview.notebookbar .icon-view-item-container img')
```

Comment:
Makes sense, I just lowered the priority of this in case, and probably good to start interacting with the UI only after we settled down.